Documentation/Helm-Charts/operator-chart.md

```diff
@@ -74,6 +74,7 @@ The following tables lists the configurable parameters of the rook-operator char
 | `csi.cephFSFSGroupPolicy`           | Policy for modifying a volume's ownership or permissions when the CephFS PVC is being mounted                                                    | `ReadWriteOnceWithFSType`                                 |
 | `csi.nfsFSGroupPolicy`              | Policy for modifying a volume's ownership or permissions when the NFS PVC is being mounted                                                       | `ReadWriteOnceWithFSType`                                 |
 | `csi.logLevel`                      | Set logging level for csi containers. Supported values from 0 to 5. 0 for general useful logs, 5 for trace level verbosity.                      | `0`                                                       |
+| `csi.sidecarLogLevel`                      | Set logging level for cephCSI containers. Supported values from 0 to 5. 0 for general useful logs (the default), 5 for trace level verbosity.                      | `0`                                                       |
```

Comment:
`Set logging level for cephCSI containers` to `Set logging level for cephCSI sidecar containers`